packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -225,7 +225,7 @@ const getProps = ({
     buttonPaddingRight: isIconOnly
       ? makeSpace(0)
       : makeSpace(theme.spacing[buttonPadding[size].right]),
-    text: size === 'xsmall' ? childrenString?.trim().toUpperCase() : childrenString?.trim(),
+    text: childrenString?.trim(),
```

Comment:
I thought we wanted the text to be upper case for xsmall size? Has this changed in design?

Reply:
Yes this was changed, I discussed with RK. We have reverted this to not have uppercase.

Reply:
This was already done on design at the time of brand refresh.